components/Article.js

```diff
@@ -0,0 +1,47 @@
+import { PrismicLink, PrismicText } from "@prismicio/react";
+import { PrismicNextImage } from "@prismicio/next";
+import * as prismicH from "@prismicio/helpers";
+import { getExcerpt, findFirstImage, dateFormatter } from "../utils";
+import { Heading } from "../components/Heading";
+
+export const Article = ({ article }) => {
+  const featuredImage =
+    (prismicH.isFilled.image(article.data.featuredImage) &&
+      article.data.featuredImage) ||
+    findFirstImage(article.data.slices);
+  const date = prismicH.asDate(
+    article.data.publishDate || article.first_publication_date
+  );
+  const excerpt = getExcerpt(article.data.slices);

```

Comment:
Style nitpick: We could add some whitespace to make this easier to read (I realize the original version was compact like this as well 🙃).

```ts
import { PrismicLink, PrismicText } from "@prismicio/react";
import { PrismicNextImage } from "@prismicio/next";
import * as prismicH from "@prismicio/helpers";

import { getExcerpt, findFirstImage, dateFormatter } from "../utils";
import { Heading } from "../components/Heading";

export const Article = ({ article }) => {
  const featuredImage =
    (prismicH.isFilled.image(article.data.featuredImage) &&
      article.data.featuredImage) ||
    findFirstImage(article.data.slices);

  const date = prismicH.asDate(
    article.data.publishDate || article.first_publication_date
  );

  const excerpt = getExcerpt(article.data.slices);

  // ...
```

Reply:
Great feedback, just updated it @angeloashmore . Let me know if that's what you had meant or if any other changes should  happen.